src/Components/CloseEmailButton/index.jsx

```diff
@@ -0,0 +1,12 @@
+import { Link } from "react-router-dom"
+
+function CloseEmailButton (){
+    return (
+        <div className="grid justify-items-end pt-4">
+            <button className="px-4 py-1.5 bg-gray-500 text-white rounded hover:bg-gray-600">
```

Comment:
You probably don't need this button tag anymore, just move the classes onto the Link and go with that